### DISCUSSION TITLE
Insider Docker Setup Issues.

### DISCUSSION THREAD

**Discussion by VR2ualize** [squidfunk/mkdocs-material#4171] [Q&A]
Hi :-)

I got an issue and a question about the insider setup with docker.

I get to the following in the setup. https://squidfunk.github.io/mkdocs-material/insiders/getting-started/#with-docker

---

➜  ~ docker login -u VR2ualize -p ghp_PXMXXXXXXXXXXXXXXXXXXXvix ghcr.io

WARNING! Using --password via the CLI is insecure. Use --password-stdin.
Login Succeeded

➜  ~ docker pull ghcr.io/vr2ualize/mkdocs-material-insiders                       

Using default tag: latest
Error response from daemon: manifest unknown
➜  ~ 

---

Anyone else seen this error before or know how I could possible trouble shoot this?

Thanks  :-)

**Comment by VR2ualize** [comment]
Hi .. just having another look at this, and I think I may be missing a step. ( step 5 )

https://squidfunk.github.io/mkdocs-material/insiders/getting-started/#with-docker

" 5 - [Create a new release](https://docs.github.com/en/github/administering-a-repository/managing-releases-in-a-repository#creating-a-release) to build and publish the Docker image "

I did the new release, but I may have missed "publish the Docker image"  bit, I assume I need to publish it and sign up to dockerhub or something to have a private registry, I dont know, a bit new to me :-)

**Reply by blueswen** [reply]
Maybe you can check the ```Build Docker image``` job status in your repository actions? According to the action yaml, the image will be push to your repository packages like a private registry after build and check.

**Comment by VR2ualize** [comment]
The problem must be between item 5 and 6 here.
https://squidfunk.github.io/mkdocs-material/insiders/getting-started/#with-docker

When I do this "docker pull ghcr.io/vr2ualize/mkdocs-material-insiders" 

I get
 "Using default tag: latest  
Error response from daemon: manifest unknown"

Is it possible that in step 5 after I create the new release that I need to do something extra to "publish docker image" ?

**Reply by squidfunk** [reply]
I'm on vacation until mid-August, so unfortunately I can't debug this issue right now but it sounds like either your login did not succeed, or the image was not successfully built and pushed to your private registry. [See this StackOverflow answer](https://stackoverflow.com/a/63474733).
